packages/amplify-appsync-simulator/src/velocity/util/opensearch-utils.ts

```diff
@@ -0,0 +1,631 @@
+/**
```

Comment:
should the class be named OpensearchUtils to match the file?

Reply:
Well AppSync still uses the "ElasticSearch" terminology for it's [utility methods](https://docs.aws.amazon.com/appsync/latest/devguide/transformation-helpers-in-utils-transform.html) that we're mocking here, so can I keep it as is to avoid more confusion?